common/web/types/src/package/kmp-json-file.ts

```diff
@@ -74,3 +81,27 @@ export interface KmpJsonFileStartMenuItem {
   icon?: string;
   location?: string;
 }
+
+export interface KmpJsonFileExample {
+  /**
+   * BCP 47 identifier for the example
+   */
+  id: string;
```

Comment:
I see this is the language ID that the example is for.
If a keyboard package contains multiple keyboards, should we also specify what keyboard ID the example is for?

Reply:
The examples array is a sub-object of the keyboard object.